### PR TITLE
added finished message to increase UX

### PR DESCRIPTION
**Pull request by ThorstenBux**
Just added a single line to the artoolkit5-setenv file to show that the environment variable was set successfully.

### CODE REVIEW

File: share/artoolkit5-setenv [hunk 1]
```diff
@@ -11,3 +11,5 @@ if [ "`uname`" = "Darwin" ]
 then
 	defaults write ~/.MacOSX/environment ARTOOLKIT5_ROOT -string "${ARTOOLKIT5_ROOT}"; plutil -convert xml1 ~/.MacOSX/environment.plist
 fi
+
+echo “*** Successfully set ARTOOLKIT5_ROOT to ${ARTOOLKIT5_ROOT} ***”
```
Comment (philip-lamb) [inline share/artoolkit5-setenv]: 1. Replace curly quotes with straight quotes.
2. Finish file with a LF.

Comment (ThorstenBux) [inline share/artoolkit5-setenv]: Done

**Comment by ThorstenBux** [comment]
Added the single quotes and the LF.
Squashed the commits and that seemed to have cased that our conversation is now hidden.

File: share/artoolkit5-setenv [hunk 2]
```diff
@@ -11,3 +11,5 @@ if [ "`uname`" = "Darwin" ]
 then
 	defaults write ~/.MacOSX/environment ARTOOLKIT5_ROOT -string "${ARTOOLKIT5_ROOT}"; plutil -convert xml1 ~/.MacOSX/environment.plist
 fi
+
+echo ‘*** Successfully set ARTOOLKIT5_ROOT to ${ARTOOLKIT5_ROOT} ***’
```
Comment (philip-lamb) [inline share/artoolkit5-setenv]: Still using curly quotes... only now curly single quotes. :-)

Comment (ThorstenBux) [inline share/artoolkit5-setenv]: Maybe we should try talking in ASCII code ;-). 
Are these now the correct ones?

File: share/artoolkit5-setenv [hunk 3]
```diff
@@ -11,3 +11,5 @@ if [ "`uname`" = "Darwin" ]
 then
 	defaults write ~/.MacOSX/environment ARTOOLKIT5_ROOT -string "${ARTOOLKIT5_ROOT}"; plutil -convert xml1 ~/.MacOSX/environment.plist
 fi
+
+echo `*** Successfully set ARTOOLKIT5_ROOT to ${ARTOOLKIT5_ROOT} ***`
```
Comment (philip-lamb) [inline share/artoolkit5-setenv]: These are backticks. Use straight double-quotes.

**Pull request closed without merging**